package/system/uci/patches-host/001-fix-langjump-compile-error-host-build.patch

```diff
@@ -0,0 +1,11 @@
+--- a/uci_internal.h
++++ b/uci_internal.h
+@@ -185,7 +185,7 @@ struct uci_backend _var = {		\
+  */
+ #define UCI_HANDLE_ERR(ctx) do {	\
+ 	DPRINTF("ENTER: %s\n", __func__); \
+-	int __val = 0;			\
```

Comment:
Add gcc version and error message to the commit description.

Reply:
This is the message I get if the patche is ont applied on my debian jessie compilation host

> make[5]: Entering directory '/home/feckert/workspace/lede/LDM-master-64/build/openwrt/build_dir/hostpkg/uci-2019-11-14-8dd50da2'
> [  5%] Building C object CMakeFiles/uci.dir/libuci.c.o
> [ 11%] Building C object CMakeFiles/uci.dir/file.c.o
> In file included from /home/feckert/workspace/lede/LDM-master-64/build/openwrt/build_dir/hostpkg/uci-2019-11-14-8dd50da2/file.c:34:0:
> /home/feckert/workspace/lede/LDM-master-64/build/openwrt/build_dir/hostpkg/uci-2019-11-14-8dd50da2/file.c: In function 'uci_import':
> /home/feckert/workspace/lede/LDM-master-64/build/openwrt/build_dir/hostpkg/uci-2019-11-14-8dd50da2/uci_internal.h:188:6: error: variable '__val' might be clobbered by 'longjmp' or 'vfork' [-Werror=clobbered]
>   int __val = 0;   \
>       ^
> /home/feckert/workspace/lede/LDM-master-64/build/openwrt/build_dir/hostpkg/uci-2019-11-14-8dd50da2/file.c:653:2: note: in expansion of macro 'UCI_HANDLE_ERR'
>   UCI_HANDLE_ERR(ctx);
>   ^~~~~~~~~~~~~~
> cc1: all warnings being treated as errors
> CMakeFiles/uci.dir/build.make:75: recipe for target 'CMakeFiles/uci.dir/file.c.o' failed
> make[5]: *** [CMakeFiles/uci.dir/file.c.o] Error

Reply:
Whatever, it needs to go into commit description, including the gcc version. I don't know what is used in Debian Jessie from top of my head.

Reply:
I have update the commit with your required changes.
https://github.com/openwrt/openwrt/pull/2583/commits/374720b0265753e239e58134ff519fe1d4994ba2